### PR TITLE
Name the test tools `contest`

### PR DESCRIPTION
**Pull request by utam0k**
Until now our testing tools have not had names, but we will give them names to make them easier to understand.

### CODE REVIEW

**Comment by YJDoc2** [comment]
Hey, I'll take a look at this around the weekend. In the meantime, May I ask you to check if we need any changes in our documentation for this?

File: .gitmodules [hunk 1]
```diff
@@ -1,5 +1,3 @@
-[submodule "integration_tests/oci-runtime-tests"]
+[submodule "tests/oci-runtime-tests/src/github.com/opencontainers/runtime-tools"]
 	path = tests/oci-runtime-tests/src/github.com/opencontainers/runtime-tools
 	url = https://github.com/opencontainers/runtime-tools.git
-	ignore = dirty
```
Comment (YJDoc2) [inline .gitmodules:4]: I think we should still keep the `ignore = dirty` part ? Is it not needed?

**Review by YJDoc2 — COMMENTED** [review]
I feel moving the test_framework outside the contest, i.e. under the path `tests/test_framework` would be better, as then we can re-use it for other tests such as podman rootless and wasm tests when we add them.
---

@utam0k Overall looks nice, just have few comments, please take a look at them.

**Comment by YJDoc2** [comment]
@utam0k :eyes:

**Comment by utam0k** [comment]
Sorry I don't find out the time to address it 🙏

**Comment by YJDoc2** [comment]
@utam0k , overall looks ok, but can you take a look at the remaining comments above when you get time? Thanks!

File: .gitmodules (hunk 1 above)
Comment (utam0k) [reply .gitmodules:4]: I have already addressed it.

**Comment by utam0k** [comment]
@YJDoc2 Thanks for your quick review 🙏  Is there anything I can address?

**Review by YJDoc2 — APPROVED** [review]

**Pull request merged**